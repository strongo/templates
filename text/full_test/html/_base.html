<!DOCTYPE html>
<html>
<head>
    <title>{% block head_title %}BLOCK head_title{% endblock %}</title>
</head>
<body>
{% block body %}
    {% block header %}
        <div id="header">
            <h1>{% block h1_content %}BLOCK h1_content{% endblock %}</h1>
        </div>
    {% endblock %}
{% endblock %}
</body>
</html>