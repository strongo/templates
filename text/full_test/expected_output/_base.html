<!DOCTYPE html>
<html>
<head>
    <title>BLOCK head_title</title>
</head>
<body>


        <div id="header">
            <h1>BLOCK h1_content</h1>
        </div>


</body>
</html>